lib/game.rb

```diff
@@ -1,15 +1,66 @@
 # frozen_string_literal: true
 
-require_relative 'board'
-
 class Game
-  attr_accessor :board
+  attr_reader :current_player
 
-  def initialize
-    @board = Board.new
+  def initialize(board, players, console)
+    @board = board
+    @players = players
```

Comment:
It looks like `@players` is only ever being used to access `player2`. Couldn't we just initialize this class with two `Player` objects?

Reply:
ohhhhh

Reply:
We can still keep the `Players` class to manage the turn order and player types. Maybe it can return a method `get_players` that has the two players initialized in the desired order, and inject those two players into the `Game` class.

Reply:
Updates pushed but not sure if it did it right? Also, Rubocop won't let me preface a method name with `get_` so I called it `initial_order`